pkg/crc/machine/machine.go

```diff
@@ -110,12 +144,18 @@ func Start(startConfig StartConfig) (StartResult, error) {
 
 		result.Status = vmState.String()
 	} else {
-		host, err := libMachineAPIClient.Load(machineConfig.Name)
+		host, err := libMachineAPIClient.Load(startConfig.Name)
 		if err != nil {
 			result.Error = err.Error()
 			return *result, errors.Newf("Error loading host: %v", err)
 		}
 
+		bundleName, _ := host.Driver.GetBundleName()
```

Comment:
Here we are ignoring the `error` which we should not because when I tried stopped and started instance with different version then gave memory dump because we ignored the error.

Reply:
The error is ignored on purpose, because fillClusterConfig is supposed to handle bundleName being "". There is a case when fillClusterConfig fails to return an error, which probably causes the issue you had:
```
diff --git a/pkg/crc/machine/machine.go b/pkg/crc/machine/machine.go
index 0f22d29..2541f7d 100644
--- a/pkg/crc/machine/machine.go
+++ b/pkg/crc/machine/machine.go
@@ -72,7 +72,7 @@ func loadCrcBundleInfo(startConfig *StartConfig, bundleName string) (*bundle.Crc
                        return nil, errors.Newf("Error getting bundle Metadata %v", err)
                }
                logging.Warnf("Could not read bundle name from VM, falling back to %s", bundles[0])
-               crcBundleMetadata, err = bundle.GetCachedBundleInfo(bundles[0])
+               return bundle.GetCachedBundleInfo(bundles[0])
        } else {
                crcBundleMetadata, err = bundle.GetCachedBundleInfo(bundleName)
                if err != nil {
``` 
It's debatable whether it's a good idea to "hide" error handling this way :) However, since we agreed on error out in this release when the bundle in use is not the one we expect, this code will be gone when I update the PR.

Reply:
@cfergeau since the code around getting the bundle name from the cached directory is already removed do we need to still hide the error here?

Reply:
Forgot to change it, updated now.